.github/workflows/functional-test.yaml

```diff
@@ -96,7 +96,6 @@ jobs:
 
       - name: Install grpcurl
         run: |
-          go get github.com/fullstorydev/grpcurl/...
```

Comment:
apparently how `go` works now, you no longer need to do `go get` (it yells at you + fails). This change has been tested to work